filament/README.md

```diff
@@ -84,7 +84,7 @@ Copy your platform's Makefile below into a `Makefile` inside the same directory.
 ### Linux
 
 ```
-FILAMENT_LIBS=-lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v
+FILAMENT_LIBS=-lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v -libl
```

Comment:
smol-v shouldn't be necessary when Vulkan support is off right?

Reply:
It shouldn't, be our releases are compiled with Vulkan support, so it's required here.

Reply:
To clarify, only desktop releases should have Vulkan included by default.

Reply:
Just to be clear our Android releases are definitely not compiled with Vulkan support but the desktop build is, so this comment is the README is fine as-is.